Backend/routes/routes.py

```diff
@@ -1,4 +1,69 @@
 from fastapi import APIRouter
+
+
+from Backend.model.model import User
+from Backend.config.database import collection, db
+from Backend.schemas.schema import user_addition_serializer
```

Comment:
Probably we can name it as user_list_serializer

Reply:
yes changed to user_list_serializer